openedx/features/course_experience/__init__.py

```diff
@@ -58,3 +59,15 @@ def course_home_url_name(course_key):
         return 'openedx.course_experience.course_home'
     else:
         return 'info'
+
+
+class CourseHomeMessages(UserMessageCollection):
```

Comment:
Nit: I'm not sure about this name. It doesn't reflect all messages shown on the course home page. I'd been thinking ``CourseWelcomeMessages``, but I guess it doesn't reflect all welcome messages either. If we can't think of anything better than I'm fine with you going with whichever one you're comfortable with.